mr/build.gradle

```diff
@@ -68,8 +67,7 @@ project(':iceberg-mr') {
     testImplementation libs.avro.avro
     testImplementation libs.calcite.core
     testImplementation libs.kryo.shaded
-    testImplementation platform(libs.jackson.bom)
-    testImplementation "com.fasterxml.jackson.core:jackson-annotations"
+    testImplementation enforcedPlatform(libs.jackson212.bom)
```

Comment:
according to https://github.com/apache/hive/pull/4562 the Jackson version was switched to 2.9.4

Reply:
I'm actually wondering how this worked in the first place with Jackson 2.14 and Hive 2.3.9

Reply:
it was bumped to 2.12.0 soon https://github.com/apache/hive/commit/2469d9e7d9ffa66f1d213539bd42243125ba930c

and it **should** be compatible with 2.12 onwards ...